packages/icons/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@codesandbox/icons",
+  "version": "0.0.1",
+  "description": "Icons designed for the codesanbox app",
+  "main": "react/index",
+  "scripts": {
+    "storybook": "start-storybook -p 6007",
+    "build": "node scripts/build"
+  },
+  "license": "MIT",
+  "devDependencies": {
+    "@storybook/react": "5.2.0-beta.28",
+    "lodash.camelcase": "^4.3.0",
+    "lodash.upperfirst": "^4.3.1",
+    "prettier": "^1.18.2",
```

Comment:
Prettier recommends using an exact version

https://prettier.io/docs/en/install.html

> We recommend pinning an exact version of prettier in your `package.json` as we introduce stylistic changes in patch releases.

Reply:
no feedback on the code please (yet), this is a draft 😇

Reply:
Sorry for my enthusiasm 😁😇